### PR TITLE
Add CheckForXinetd from openSUSE/rpmlint-checks (#227).

### PR DESCRIPTION
**Pull request by marxin**

### CODE REVIEW

File: rpmlint/descriptions/CheckForXinetd.toml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+obsolete-xinetd-requirement="""
+In systemd based distributions xinetd has become obsolete.
```
Comment (scarabeusiv) [inline rpmlint/descriptions/CheckForXinetd.toml]: Could you please mention socket activated services here?

File: rpmlint/configdefaults.toml [hunk 2]
```diff
@@ -2,6 +2,7 @@
 Checks = [
     "AppDataCheck",
     "BinariesCheck",
+    "CheckForXinetd",
```
Comment (scarabeusiv) [inline rpmlint/configdefaults.toml]: This one should not be enabled by default.

File: rpmlint/checks/CheckForXinetd.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+from rpmlint.checks.AbstractCheck import AbstractCheck
+
+
+class CheckForXinetd(AbstractCheck):
+    def __init__(self, config, output):
+        super().__init__(config, output)
```
Comment (scarabeusiv) [inline rpmlint/checks/CheckForXinetd.py]: This is noop and can be ommited (the whole init).

File: rpmlint/descriptions/CheckForXinetd.toml (hunk 1 above)
Comment (marxin) [reply rpmlint/descriptions/CheckForXinetd.toml]: Sure, please provide a wording? I'm not so much familiar with `systemd`.

Comment (scarabeusiv) [reply rpmlint/descriptions/CheckForXinetd.toml]: Something like:

Xinetd is obsolete by systemd socket activated services.
Please stop using xinetd and switch to socket activation
from systemd.

Comment (marxin) [reply rpmlint/descriptions/CheckForXinetd.toml]: Thanks, done.

**Pull request merged**